crates/llms/src/anthropic/mod.rs

```diff
@@ -155,7 +155,7 @@ impl Config for AnthropicConfig {
     }
 
     fn url(&self, path: &str) -> String {
-        format!("{}/{}", self.api_base(), path)
```

Comment:
I found out, via a tokio error that we were technically sending `"https://api.anthropic.com/v1//messages"` across the wire and anthropic was okay with it.